sdk/appconfiguration/azure-data-appconfiguration/TROUBLESHOOTING.md

```diff
@@ -31,7 +31,8 @@ Refer to the instructions in this reference document on how to [configure loggin
 
 Reviewing the HTTP request sent or response received over the wire to/from the Azure App Configuration service can be 
 useful in troubleshooting issues. To enable logging the HTTP request and response payload, the [ConfigurationClient][configuration_client] 
-can be configured as shown below:
+can be configured as shown below and set an environment variable [AZURE_LOG_LEVEL][azure_log_level] in your machine to 
```

Comment:
This only needs to be set for the environment variable version, if you set it as part of the client builder it isn't needed.